geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/MinMax.scala

```diff
@@ -29,6 +29,9 @@ class MinMax[T] private (val attribute: Int, private [stats] var hpp: HyperLogLo
                         (implicit val defaults: MinMax.MinMaxDefaults[T], ct: ClassTag[T])
     extends Stat with LazyLogging with Serializable {
 
+  this.minValue = defaults.max
+  this.maxValue = defaults.min
```

Comment:
Maybe not anymore, it was fixing them not being initialized when we were copying when this was a case class.